docs/configuration.md

```diff
@@ -492,33 +492,26 @@ Use [globbing syntax](https://en.wikipedia.org/wiki/Glob_(programming)) for wild
 
 ## Baseline
 
-### `baseline` &lt;`command`&gt;
+### `with-baseline` &lt;`committish`&gt;
 
-Default: `false`
-Command line: `dotnet stryker baseline`
-Config file: `"baseline": { }`
+Default: `false`  
+Command line: `with-baseline feat-2`  
+Config file: `"baseline": { }`  
 
-Enabling `with-baseline` saves the mutation report to a storage location such as the filesystem. The mutation report is loaded at the start of the next mutation run. Any changed source code or unit test results in a reset of the mutants affected by the change. For unchanged mutants the previous result is reused. This feature expands on the [since](#since-flag-committish) feature by providing you with a full report after a partial mutation testrun.
+Enabling `baseline`, alias `with-baseline`, saves the mutation report to a storage location such as the filesystem. The mutation report is loaded at the start of the next mutation run. Any changed source code or unit test results in a reset of the mutants affected by the change. For unchanged mutants the previous result is reused. This feature expands on the [since](#since-flag-committish) feature by providing you with a full report after a partial mutation testrun. The target of the baseline is a `committish` passed as an extra argument. The target determines which committish is used to establish or locate a baseline.
```

Comment:
Why does not still need to be a committish? We are trying to move away from git for baseline.